packages/mobile/src/onboarding/contacts/ImportContactsScreen.tsx

```diff
@@ -52,29 +95,83 @@ class ImportContactScreen extends React.Component<Props, State> {
     })
   }
 
+  onFinish = () => {
+    navigate(Screens.OnboardingSuccessScreen)
+  }
+
+  renderImportStatus = () => {
+    const { t, importContactsProgress, matchedContacts } = this.props
+    const { status, total } = importContactsProgress
```

Comment:
Nit: You can probably go for the double destructure of `status` and `total` here if you arent using `importContactsProgress` anywhere else